src/server/frontend_wayland/text_input_v1.cpp

```diff
@@ -361,6 +362,7 @@ void TextInputV1::activate(wl_resource *seat_resource, wl_resource *surface)
     {
         on_new_input_field = true;
         pending_state.emplace();
+        pending_state_used = false;
```

Comment:
This is where we can break the hypothetical invariant, but I'm not sure what it achieves